platform/linux-generic/odp_packet_io.c

```diff
@@ -1617,6 +1634,46 @@ int odp_pktin_recv_tmo(odp_pktin_queue_t queue, odp_packet_t packets[], int num,
 	}
 }
 
+static int odp_pktin_recv_mq_tmo_select(const odp_pktin_queue_t queues[],
```

Comment:
Internal functions should not use odp_ prefix.

Reply:
Will fix.